app/components/modal/profile/user.profile.controller.js

```diff
@@ -0,0 +1,41 @@
+(
+    function () {
+        'use strict';
+
+        angular.module('FoodOrderingApp')
+            .controller('UserProfileController', UserProfileController);
+
+        UserProfileController.$inject = ['$sessionStorage', 'balance', '$uibModalInstance','$uibModal', 'RestaurantService', 'OrderService', '$log'];
+
+        function UserProfileController($sessionStorage, balance, $uibModalInstance,$uibModal, RestaurantService, OrderService, $log) {
+            var vm = this;
+            vm.user = $sessionStorage.firstName;
+            vm.balance = balance;
+
+            vm.closeModal = function(){
+                $uibModalInstance.close();
+            }
+
```

Comment:
I don't think there is any use of $uibModalInstance.close(); in this controller, I think LogoutController can only have $uibModalInstance. Is there any use of closeModal in this controller ?